images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -0,0 +1,76 @@
+################################################################################
+##  File:  Install-Msys2.ps1
+##  Desc:  Install Msys2 and 64-bit gcc, cmake, & llvm (32-bit commented out)
+################################################################################
+
+# References
+# https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml
+# https://packages.msys2.org/group/
+
+$origPath = $env:PATH
+$gitPath  = "C:\Program Files\Git"
+
```

Comment:
Perhaps use `$env:ProgramFiles` instead of `C:\Program Files`?

Reply:
Done, thank you!